packages/dashboard/src/components/starter/next-steps/next-steps.tsx

```
@@ -0,0 +1,81 @@
+  return (
+    <div class="container container-purple container-center">
+      <h2>
+        Time for a
+        <br />
+        <span class="highlight">qwik intro</span>?
+      </h2>
+      <div class={styles.gettingstarted}>
+        <div
+          class={styles.intro}
+          dangerouslySetInnerHTML={
+            GETTING_STARTED_STEPS[gettingStartedStep.value].message
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)